src/helpers/db.ts

```diff
@@ -0,0 +1,8 @@
+import { Config, JsonDB } from 'node-json-db';
+
+export const db = new JsonDB(new Config('todoDB', true, true, '/'));
+
+export const initilizeDB = async () => {
+  if (!(await db.exists('/categories'))) db.push('/categories', []);
```

Comment:
I wonder why you are checking this if the db exists while JSON db by default gets created when the application runs. Maybe there is something I am missing, can enlighten me on this., because I think there will be no scenario this db will not be there!